wagtail/models/i18n.py

```diff
@@ -210,6 +220,26 @@ def check(cls, **kwargs):
                 )
             )
 
+        if is_translation_model and not any(
+            isinstance(constraint, models.UniqueConstraint)
+            and set(constraint.fields) == {"translation_key", "locale"}
+            for constraint in constraints
+        ):
+            errors.append(
+                checks.Warning(
+                    "{}.{} is missing a UniqueConstraint for the fields: {}. Consider using models.UniqueConstraint.".format(
+                        cls._meta.app_label, cls.__name__, unique_constraint_fields
+                    ),
+                    hint="Add models.UniqueConstraint(fields={}, name='unique_{}') to {}.Meta.constraints.".format(
```

Comment:
We'll need to make it so that the suggested `name` is unique and follows what we plan to do in the future, i.e. in the format of `unique_translation_key_locale_%(app_label)s_%(class)s`. This is missing the `app_label` and `class`.